docs_guidelines/apidocs/tutorials.guidelines.rst

```diff
@@ -0,0 +1,110 @@
+####################
+Tutorials Guidelines
+####################
+
+1. Get the user started
+=======================
+
+1.1 Lower the barrier of entry 
+------------------------------
+
+Lower the barrier of entry as much as possible. Assume the user does not know much about what is
+covered in the tutorial. If we are expecting users to know something, make it explicit and link them
+to the needed explanations (See :ref:`tutorial-guidelines-minimum-explanation`) Avoid jargon as much as possible. Provide a minimum
+necessary explanation when the jargon is introduced. (See :ref:`tutorial-guidelines-minimum-explanation`)
+ 
+1.2 Provide setup / prerequisites
+---------------------------------
+
+1.3 Add sufficient comments to code
+-----------------------------------
+
+2. Provide a complete picture before they start
+===============================================
+
+2.1 Provide overview at the start
+---------------------------------
+
+Summarize what the tutorial is about in the first paragraph, ideally the first sentence
+
+    "In this tutorial, you will..."
+
+2.2 Provide context
+-------------------
+
+Add a context for why you want to do this tutorial? Or who is this tutorial for?
+
+2.3 Summarize at the end
+------------------------
+
+Summarize what the user has accomplished at the end of the tutorial
+
+    "You have built a secure, three-layer hylomorphic stasis engine…" Describe (and admire, in a 
+    mild way) what your learner has accomplished (note - not: “you have learned…”)
```

Comment:
I think in this sentence we could elaborate a bit more about not using condescending language. So explain that they shouldn't make assumptions about what the audience has achieved (i.e. explain why not to use "you have learned..") or make assumptions about what they already know or are able to understand (so avoid words like, "obviously", "trivial", or "as you know")

Reply:
When I initially read that point (taken verbatim from [Diataxis](https://diataxis.fr/tutorials/#the-language-of-tutorials)) what I understood was mostly to think of the user as an independent being doing things on their own (in a "you can do it" sense instead of "you are alone with the lions, fight for your life") instead of a pupil that is listening to our lesson ("you have learnt ..."). It's the same reason we use "you" instead of "we", that is, to highlight the agency of the user.

On the other side, you can't admire too much that, for example, a user has built a basic circuit with Qiskit because it would sound as if we were almost mocking them.

However I agree that we'd better explicitly tell writers to avoid any kind of condescending language like "obviously", "trivial" or "as you (should) know". This kind of language can only discourage people from using Qiskit.